Codeinwp/otter-blocks
# Fix/formbricks

Closes #2393.


### Summary


This updates the Formbricks package to make sure the notice still shows up.

### Screenshots 

----

### Test instructions


- Install Otter plugin
- Modify otter_blocks_install unix timestamp value to be at least 2 days old in wp_options table
- Check if Survey appears on Otter pages
- You can also add `&formbricksDebug=true` to the URL of the Otter dashboard to see logs in the console.
